### ISSUE TITLE
DBFtrim Info Display: MaxIntD side effects

### ISSUE THREAD

**Issue by yakra** [yakra/tmtools#23]
Sometimes the info display shows a ` <- ` in the Data column for type N/F fields when extra 0s were not trimmed.
There does not seem to be any effect on actual file output.

• Only observed in **ROADS_ACF.yOrig.dbf**
```
ah_blm		F	31	12	0.86811049 <- 0.86811049
ah_elm		F	31	12	0.54514911 <- 0.54514911
ah_length	F	31	11	0.03254181 <- 0.03254181
ah_seg_num	F	31	3	115 <- 115.000000000000000
```
All of the first 3 fields had "0000000" trimmed

**Comment by yakra** [comment]
**dbf.cpp**
*59* `oDBF.fArr[i].MinEx0 = 255;`

**Comment by yakra** [comment]
`10.103` @ offset 236687, record # foo
`0.03254181` @ offset 295632, record # bar

**Comment by yakra** [comment]
> When `10.103` is encountered, there's no way to know a longer string will be encountered later.

Ergo,
> Allowing Info Display to a "full-width" value would require: ...
 • reworking how the null terminator is stored.

...Because we've got to wait till later, when *MinEx0* is known.
Inserting a new null terminator *MinEx0* places from the end of the string is problematic: How can I be sure it leaves the proper amount of decimal places? A file can conceivably contain strings with different counts of decimal places, E.G. `1.5`, `1.25`, `1.125`, `1.0625`, etc.
I hadn't seen this out in the wild, but something told me it's wise to allow for this possibility.

> A possible workaround is to store a *mvIntD* in reserved bytes, and prepend *MaxIntD-mvIntD* characters ('#' or something) to MaxVal in the InfoDisplay.

Nope. I found a case of "different decimal counts" in **txdot-2015-roadways_48113.dbf**.
IE, *MaxVal* for `Shape_Leng` is `0.0020730099`, which takes up all 12 bytes of the original field size.
`Shape_Leng` also contains a value of `10.647177   `, which also fits within those 12 bytes, due to having fewer decimal digits.
Thus, *MaxIntD-mvIntD* == 1, and one '#' is prepended. "#0.0020730099" implies a 13 byte field size, not the 12 bytes allocated in the original file.

So the actual fix is even simpler:
Don't worry about *mvIntD* at all, and just add *len-strlen(MaxVal)* characters.
